### PR TITLE
Make approved features non-experimental

### LINKED ISSUE #11081: Certain errors are not shown when all experimental features are disabled
When all experimental features are disabled, certain errors are not shown at all. It also seems that the "PSErrorView" experimental feature is always enabled, even after explicitly disabling it.

This does **not** repro in Preview.5

```
PS> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      7.0.0-preview.5
PSEdition                      Core
GitCommitId                    7.0.0-preview.5
OS                             Microsoft Windows 10.0.19516
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

PS C:\Users\adityap> $EnabledExperimentalFeatures
PS C:\Users\adityap> "".$dsfs()
Method invocation failed because [System.String] does not contain a method named ''.
At line:1 char:1
+ "".$dsfs()
+ ~~~~~~~~~~
+ CategoryInfo          : InvalidOperation: (:) [], RuntimeException
+ FullyQualifiedErrorId : MethodNotFound
```

# Steps to reproduce

```powershell
PS C:\Windows\System32> $EnabledExperimentalFeatures
PS> "".$dsfs()
PS> $Error[0]

PS> $Error[0] | fl *

PS> $Error[0] | fl * -Force

PSMessageDetails      :
Exception             : System.Management.Automation.RuntimeException: Method invocation failed because [System.String] does not contain a method named ''.
                           at CallSite.Target(Closure , CallSite , String , Object )
                           at System.Dynamic.UpdateDelegates.UpdateAndExecute2[T0,T1,TRet](CallSite site, T0 arg0, T1 arg1)
                           at System.Management.Automation.Interpreter.DynamicInstruction`3.Run(InterpretedFrame frame)
                           at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame frame)
TargetObject          :
CategoryInfo          : InvalidOperation: (:) [], RuntimeException
FullyQualifiedErrorId : MethodNotFound
ErrorDetails          :
InvocationInfo        : System.Management.Automation.InvocationInfo
ScriptStackTrace      : at <ScriptBlock>, <No file>: line 1
PipelineIterationInfo : {}
```

# Expected behavior

```none
Error is shown in default error view
```

# Actual behavior

```none
No error shown
```

# Environment data

```none

Name                           Value
----                           -----
PSVersion                      7.0.0-daily.20191114
PSEdition                      Core
GitCommitId                    7.0.0-daily.20191114
OS                             Microsoft Windows 10.0.19516
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

### LINKED ISSUE #11302: Decide on which experimental features will remain in experimental in 7.0
Going into RC, we have to make a decision on whether experimental features are "graduated" to stable vs. staying as experimental.

Per some conversations with the @PowerShell/powershell-committee, this is our current position on each:

## Stable

```
PSCoalescingOperators
PSErrorView
PSForEachObjectParallel
PSPipelineChainOperators
PSTernaryOperator
PSUpdatesNotification
PSWindowsPowerShellCompatibility
Microsoft.PowerShell.Utility.PSGetError
```

## Experimental

* `PSCommandNotFoundSuggestion`: the suggestion infrastructure has proven to have some serious issues that break this feature (e.g. it's not shown at all in VS Code)
* `PSImplicitRemotingBatching`: this has not had enough significant usage (both internally and externally) for us to feel confident about it
* `PSNullConditionalOperators`: this is only the null conditional member access aspect of the null conditional operators
* `PSUnixFileState`: this could still have unknown performance implications, and we still might want to change the property names on the returned information
* `Microsoft.PowerShell.Utility.PSManageBreakpointsInRunspace`: this one will likely be getting some last minute work from @TylerLeonhardt to support a feature needed in `vscode-powershell`. Given the risk of these last minute changes, we don't want to promote this to stable in 7.0.
* `PSDesiredStateConfiguration.InvokeDscResource`: this has some limitations compared to the `Invoke-DscResource` shipped in Windows PS that we'd like to address before graduating it.

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
# PR Summary

As discussed in @PowerShell/powershell-committee, these features will be out of Experimental status:

```none
PSCoalescingOperators
PSErrorView
PSForEachObjectParallel
PSPipelineChainOperators
PSTernaryOperator
PSUpdatesNotification
PSWindowsPowerShellCompatibility
Microsoft.PowerShell.Utility.PSGetError
```

Code marking them as Experimental has been removed.

## PR Context

Fix https://github.com/PowerShell/PowerShell/issues/11302
Fix https://github.com/PowerShell/PowerShell/issues/11081

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [x] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

**Review by anmenaga — CHANGES_REQUESTED** [review]
`PSWindowsPowerShellCompatibility` has to remain experimental. Primary reasons:
1) not enough flight time yet. First time it appeared only few weeks ago in Preview 6. On high level this feature changes module load behavior for some modules so it has capability to change/mess up existing user scripts/scenarios that were running fine before it. We've seen this happened with ScheduledJobs and CertificateProvider.
This needs more flight time.
2) As continuation from #1 if something goes wrong because of this feature - currently the only mechanism to turn it off is ExperimentalFeature switch.

**Comment by troymoore** [comment]
Understoodneedmoretime

File: src/Microsoft.PowerShell.Commands.Utility/commands/utility/Get-Error.cs [hunk 1]
```diff
@@ -11,7 +11,6 @@ namespace Microsoft.PowerShell.Commands
     /// <summary>
```
Comment (PoshChan) [inline src/Microsoft.PowerShell.Commands.Utility/commands/utility/Get-Error.cs]: @SteveL-MSFT, your last commit had 5 failures in `PowerShell-CI-macos`
Error occurred in test script 'test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1'
```powershell
At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1:106 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/runner/runners/2.160.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1'
```powershell
At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1:106 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/runner/runners/2.160.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1'
```powershell
At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1:106 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/runner/runners/2.160.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1'
```powershell
At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1:106 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/runner/runners/2.160.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1'
```powershell
At /Users/runner/runners/2.160.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1:106 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```

Comment (PoshChan) [inline src/Microsoft.PowerShell.Commands.Utility/commands/utility/Get-Error.cs]: @SteveL-MSFT, your last commit had 4 failures in `PowerShell-CI-windows`
Error occurred in test script 'D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Foreach-Object-Parallel.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Foreach-Object-Parallel.Tests.ps1:106 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.1\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Error occurred in test script 'D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Foreach-Object-Parallel.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Foreach-Object-Parallel.Tests.ps1:106 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.1\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Error occurred in test script 'D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Foreach-Object-Parallel.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Foreach-Object-Parallel.Tests.ps1:106 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.1\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Error occurred in test script 'D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Foreach-Object-Parallel.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Foreach-Object-Parallel.Tests.ps1:106 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.1\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```

Comment (PoshChan) [inline src/Microsoft.PowerShell.Commands.Utility/commands/utility/Get-Error.cs]: @SteveL-MSFT, your last commit had 5 failures in `PowerShell-CI-linux`
Error occurred in test script 'test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1'
```powershell
At /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1:106 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, /home/vsts/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1'
```powershell
At /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1:106 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, /home/vsts/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1'
```powershell
At /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1:106 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, /home/vsts/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1'
```powershell
At /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1:106 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, /home/vsts/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1'
```powershell
At /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Foreach-Object-Parallel.Tests.ps1:106 char:1
+ }
+ ~
Unexpected token '}' in expression or statement.
at <ScriptBlock>, /home/vsts/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```

**Comment by SteveL-MSFT** [comment]
CodeFactor issues are unrelated to my specific changes

File: src/System.Management.Automation/engine/parser/Parser.cs [hunk 2]
```diff
@@ -6721,11 +6650,6 @@ private ExpressionAst ExpressionRule(bool endNumberOnTernaryOpChars = false)
             // G      binary-expression   '?'   new-lines:opt   ternary-expression   new-lines:opt   ':'   new-lines:opt   ternary-expression
 
             // TODO: remove this if-block when making 'ternary operator' an official feature.
-            if (!ExperimentalFeature.IsEnabled("PSTernaryOperator"))
```
Comment (rjmholt) [inline src/System.Management.Automation/engine/parser/Parser.cs]: Counterpoint here: if we want a backward compatible parser/tokenizer, we should keep some kind of static flags in so that we could eventually create an implementation where you can build instances of parsers targeting different versions or feature sets.

Comment (rjmholt) [reply src/System.Management.Automation/engine/parser/Parser.cs]: Just preserving some element of the code in here will preserve enough information for us to unpick all this later.

Comment (SteveL-MSFT) [reply src/System.Management.Automation/engine/parser/Parser.cs]: Would a comment be sufficient?

```csharp
// Added in PS7.0
```

**Comment by SteveL-MSFT** [comment]
@anmenaga if this feature becomes blocking/impactful negatively for customers, we would be ok taking a change to add a setting to powershell.config.json to disable this specific feature.

File: src/System.Management.Automation/engine/Modules/ImportModuleCommand.cs [hunk 3]
```diff
@@ -53,7 +53,7 @@ public sealed class ImportModuleCommand : ModuleCmdletBase, IDisposable
         private const string ParameterSet_FQName_ViaPsrpSession = "FullyQualifiedNameAndPSSession";
         private const string ParameterSet_ViaWinCompat = "WinCompat";
         private const string ParameterSet_FQName_ViaWinCompat = "FullyQualifiedNameAndWinCompat";
-        
+
```
Comment (adityapatwardhan) [inline src/System.Management.Automation/engine/Modules/ImportModuleCommand.cs]: extra empty line?

File: src/System.Management.Automation/engine/parser/Parser.cs [hunk 4]
```diff
@@ -6098,31 +6046,12 @@ private PipelineBaseAst PipelineChainRule()
...
-                        {
-                            scanning = false;
-                        }
-
-                        break;
+                        scanning = false;
+                        continue;
 
                     case TokenKind.Ampersand:
                         // When pipeline chains are not enabled, pipelines always handle backgrounding
```
Comment (adityapatwardhan) [inline src/System.Management.Automation/engine/parser/Parser.cs]: Remove the comment too.

**Review by adityapatwardhan — CHANGES_REQUESTED** [review]

**Review by PaulHigin — APPROVED** [review]
LGTM

**Review by anmenaga — APPROVED** [review]

File: src/System.Management.Automation/engine/parser/Parser.cs (hunk 2 above)
Comment (rjmholt) [reply src/System.Management.Automation/engine/parser/Parser.cs]: Comments won't really describe the way the logic branches, but if we really want to avoid different logic (or changing the conditions to `const bool`s) then we can muddle through with something designed to look and be machine parseable for a regex like `// ps7-expfeat:PSTernaryOperator` or something

**Comment by adityapatwardhan** [comment]
@SteveL-MSFT Please resolve merge conflicts

File: src/System.Management.Automation/engine/parser/Parser.cs [hunk 5]
```diff
@@ -6721,11 +6649,6 @@ private ExpressionAst ExpressionRule(bool endNumberOnTernaryOpChars = false)
             // G      binary-expression   '?'   new-lines:opt   ternary-expression   new-lines:opt   ':'   new-lines:opt   ternary-expression
 
             // TODO: remove this if-block when making 'ternary operator' an official feature.
```
Comment (daxian-dbw) [inline src/System.Management.Automation/engine/parser/Parser.cs]: Please remove this comment

File: src/System.Management.Automation/engine/parser/Parser.cs (hunk 2 above)
Comment (daxian-dbw) [reply src/System.Management.Automation/engine/parser/Parser.cs]: If we really want to pursue the `build instances of parsers targeting different versions or feature sets` goal, then we will have to guard the to-be-removed code in a compiler definition, like `#if PSV6 ... #endif`. I can see why we want this but it won't really help unless we test those guarded code on regular basis, and eventually this will make the code difficult to maintain.

File: src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs [hunk 6]
```diff
@@ -109,24 +109,6 @@ static ExperimentalFeature()
...
-                    name: "PSForEachObjectParallel",
-                    description: "New parameter set for ForEach-Object to run script blocks in parallel"),
-                new ExperimentalFeature(
-                    name: "PSTernaryOperator",
-                    description: "Support the ternary operator in PowerShell language"),
-                new ExperimentalFeature(
-                    name: "PSErrorView",
-                    description: "New formatting for ErrorRecord"),
-                new ExperimentalFeature(
-                    name: "PSUpdatesNotification",
```
Comment (daxian-dbw) [inline src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs]: The code that uses this feature name is not cleaned up.

File: src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs [hunk 7]
```diff
@@ -109,24 +109,6 @@ static ExperimentalFeature()
                 new ExperimentalFeature(
```
Comment (daxian-dbw) [inline src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs]: For those that are kept experimental, their tests need to be declared in `test\tools\TestMetadata.json`, otherwise their tests will not be executed.
Also, for `"PSForEachObjectParallel"` and `"PSPipelineChainOperators"`, they need to be removed from that file as they become official.

**Comment by daxian-dbw** [comment]
I only reviewed the code changes related to `ternary operator` and `update notification`. Feature owners please review changes related to your features and sign off.

**Comment by daxian-dbw** [comment]
> if this feature becomes blocking/impactful negatively for customers, we would be ok taking a change to add a setting to powershell.config.json to disable this specific feature.

@SteveL-MSFT when are we going to know it becomes blocking? Do you mean after 7.0.0 GA, we can make changes in the servicing branch of 7.0.0 to add back the experimental feature declaration and checks for the win-module-compat work?

**Comment by SteveL-MSFT** [comment]
@daxian-dbw I opened a new issue https://github.com/PowerShell/PowerShell/issues/11309 which we can accept for GA

**Comment by adityapatwardhan** [comment]
@SteveL-MSFT Please fix build failures.

File: src/System.Management.Automation/engine/parser/Parser.cs (hunk 2 above)
Comment (SteveL-MSFT) [reply src/System.Management.Automation/engine/parser/Parser.cs]: Perhaps we should revisit this later.

File: src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs (hunk 6 above)
Comment (SteveL-MSFT) [reply src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs]: I thought I did a search for each one, looks like I missed this case.

**Review by adityapatwardhan — APPROVED** [review]

**Comment by adityapatwardhan** [comment]
@daxian-dbw  Please re-review.

File: test/tools/TestMetadata.json [hunk 8]
```diff
@@ -1,7 +1,5 @@
 {
   "ExperimentalFeatures": {
-    "ExpTest.FeatureOne": [ "test/powershell/engine/ExperimentalFeature/ExperimentalFeature.Basic.Tests.ps1" ],
```
Comment (daxian-dbw) [inline test/tools/TestMetadata.json]: Ideally, you should add entries for those features that are still kept as experimental, because otherwise, their tests will not be evaluated during the GA release build.
But their tests will still be run in our daily build, so that can be deferred to another PR.

**Review by daxian-dbw — APPROVED** [review]

**Comment by iSazonov** [comment]
Make sense to add CL- label?
